demisto_sdk/commands/common/native_image.py

```diff
@@ -90,6 +95,18 @@ def get_native_image_reference(self, native_image) -> Optional[str]:
 
         Returns: The docker ref
         """
+        if native_image == self.NATIVE_DEV:
+            try:
+                latest_tag = DockerImageValidator.get_docker_image_latest_tag_request(
+                    NATIVE_IMAGE_DOCKER_NAME
+                )
+                if not latest_tag:
+                    raise ValueError("Could not find the latest tag")
+            except Exception as e:
```

Comment:
Why not something like:

            try:
                latest_tag = DockerImageValidator.get_docker_image_latest_tag_request(
                    NATIVE_IMAGE_DOCKER_NAME
                )
            except Exception as e:
                logger.warning(f"Exception getting the latest tag for {native_image}: {e}")
                return None
            if not latest_tag:
                raise ValueError("Could not find the latest tag")
                return None
            return f"{NATIVE_IMAGE_DOCKER_NAME}:{latest_tag}"